habit/src/components/TextInput.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { TextInput } from 'react-native'
```

Comment:
Let's use the `native base` library's or `react-native-elements` library's text input component

Reply:
done